src/parser/reference-parser.ts

```diff
@@ -76,6 +77,20 @@ export type ExtractTypeFromStringReference<
   ? ExtractColumnType<DB, TB, RE>
   : DV
 
+export type OrderedColumnName<C extends string> =
```

Comment:
This is identical to `ExtractColumnNameFromOrderedColumnName`. Shouldn't this be just

```ts
export type OrderedColumnName<C extends string> = C | `${C} ${OrderByDirection}`
```

?

Reply:
They're very close. 
`OrderedColumnName` type-guards and returns as-is.
`ExtractColumnNameFromOrderedColumnName` extracts the column name.

The suggestion behaves differently, check [this](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcDyUAmwrEwYQgBsBXAWwDsA5AQzOAB585QZgLMBnOTmKASwoBzAHxwAvAFgAUHDjNW7LnAAGAEgDevAcIC+cTYIBmOdLpUy5cgPzoWINh24ZsUAEKIAIv1wBjGPwQFJZWNvIhoQBccBTAAG44EXDR+DIyoJCwCMioLjh4hKSUtPQATEz2jsragqIS8nAAPqqa+Pqaee5ePsD+gRTmadJIKOhYOB7efgFB9QDkNJy+c01wc9hLc0NGJBR9s0YUTKWVStw1wgA0DYpOPHy1IgAU+KXRnQXE5NR0wOWvImu+He41wBC+xV+TBEAEoNLpthQnnMAFYQAAWFDm11RGKxMIA3DJDsi0Zi4BtljiyRQKcBNjDiUjceTKXTNtS8ezfNy5ozpCSWbTOGR+DB0bzOeSRWKJZS+UA) out.